frontend/src/hooks/useSessionValid.js

```diff
@@ -105,6 +106,16 @@ function useSessionValid() {
         if (remainingTrialDays)
           userAndOrgDetails["remainingTrialDays"] = remainingTrialDays;
       }
+
+      if (getFlagStatus) {
+        const appDeploymentFlag = await getFlagStatus.fetchFeatureFlagStatus(
+          orgId,
+          csrfToken,
+          "app_deployment"
```

Comment:
@hari-kuriakose Quick doubt, should we keep the flag name as such or convert it to a env in front-end ?